COMPARE.md

```diff
@@ -0,0 +1,54 @@
+# Comparing Standard to X
+
+_Where appropriate, we compare with `divnix/paisano`, instead_.
+
+## Comparison with tools in the Nix ecosystem
+
+### flake-utils
+
+`numtide/flake-utils` is a small & lightweight utility with a focus on generating flake file _outputs_ in accordance with the packaging use case built into the Nix CLI tooling.
+
+Paisano, in turn, is an importer with a focus on _code organization_; it still plugs well into a `flake.nix` file, but also preserves its index function by keeping it clean.
+While you _can_ use it to match the schema that Nix CLI expects, it also enables more flexibility as it is not specially optimized for the Nix _package manager use case_.
+
+### flake-parts
+
+`hercules-ci/flake-parts` is a component aggregator with a focus on a flake schema that is built into the Nix CLI tooling that makes use of the NixOS module system for composability and aggregation.
+
+Paisano, in turn, is an importer with a focus on _code organization_; it still plugs well into a `flake.nix` file, but also preserves its index function by keeping it clean.
+While you _can_ use it to match the schema that Nix CLI expects, it also enables more flexibility as it is not specially optimized for the Nix _package manager use case_.
+
+To a lesser extent, Paisano is also a component aggregator for your flake outputs.
+However, in that role, it gives you back the freedom to use the output schema that best fits your problem domain.
+
+Convergence towards the Flakes output schema is provided via the harvester family of functions (`winnow`, `harvest` & `pick`).
```

Comment:
maybe we need links to the documentation for these functions, so users can understand what they are exactly, if desired?